theme/theme.go

```diff
@@ -40,21 +40,26 @@ func LightTheme() fyne.Theme {
 }
 
 // DarkTheme defines the built in dark theme colours and sizes
+// See https://www.material.io/design/color/dark-theme.html
 func DarkTheme() fyne.Theme {
 	theme := &builtinTheme{
-		background:     color.NRGBA{0x42, 0x42, 0x42, 0xff},
-		button:         color.NRGBA{0x21, 0x21, 0x21, 0xff},
-		disabledButton: color.NRGBA{0x31, 0x31, 0x31, 0xff},
-		text:           color.NRGBA{0xff, 0xff, 0xff, 0xff},
-		disabledText:   color.NRGBA{0x60, 0x60, 0x60, 0xff},
-		icon:           color.NRGBA{0xff, 0xff, 0xff, 0xff},
-		disabledIcon:   color.NRGBA{0x60, 0x60, 0x60, 0xff},
+		// background: color.NRGBA{0x12, 0x12, 0x12, 0xff}, // recommended background - very dark
+		// background:     color.NRGBA{0x1F, 0x1B, 0x24, 0xff}, // branding suggestion from the article above - very purple
+		background:     color.NRGBA{0x24, 0x24, 0x24, 0xff},
+		button:         color.NRGBA{0xff, 0xff, 0xff, 0x30},
+		hover:          color.NRGBA{0xff, 0xff, 0xff, 0x40},
+		disabledButton: color.NRGBA{0xff, 0xff, 0xff, 0x10},
+		text:           color.NRGBA{0xff, 0xff, 0xff, 0xd0},
+		disabledText:   color.NRGBA{0xff, 0xff, 0xff, 0x40},
+		icon:           color.NRGBA{0xff, 0xff, 0xff, 0xe0},
+		disabledIcon:   color.NRGBA{0xff, 0xff, 0xff, 0x60},
 		hyperlink:      color.NRGBA{0x99, 0x99, 0xff, 0xff},
 		placeholder:    color.NRGBA{0xb2, 0xb2, 0xb2, 0xff},
-		primary:        color.NRGBA{0x1a, 0x23, 0x7e, 0xff},
-		hover:          color.NRGBA{0x31, 0x31, 0x31, 0xff},
+		primary:        color.NRGBA{0x9f, 0xa8, 0xda, 0xd0}, // same as light mode
```

Comment:
This says same as light mode but they are different?

Reply:
that is true, refactor on the way later on